src/scripts/main.js

```diff
@@ -1,3 +1,51 @@
+/* eslint-disable max-len */
 'use strict';
 
-// write code here
+const list = document.querySelector('ul');
+
+const helperFn = {
+  sort(prev, next) {
+    return helperFn.convertToNumber(next.dataset.salary) - helperFn.convertToNumber(prev.dataset.salary);
+  },
+  convertToNumber(number) {
+    return +number.replace('$', '').replace(',', '.');
+  },
+  appendToList(item) {
```

Comment:
This method is redundant, there is no need to create copies of these items, you can just append their original nodes